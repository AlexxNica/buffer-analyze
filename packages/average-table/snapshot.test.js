// use story.js files as snapshots
import initStoryshots, { shallowSnapshot } from '@storybook/addon-storyshots';

initStoryshots({
  suit: 'Snapshots',
  // Shallow is needed or Highchart test will fail with error 13
  test: shallowSnapshot,
});
