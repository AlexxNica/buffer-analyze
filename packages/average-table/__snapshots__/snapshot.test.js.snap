// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots AverageTable should render a "no data" state 1`] = `
<span>
  <div
    style={
      Object {
        "width": "750px",
      }
    }
  >
    <AverageTable
      loading={false}
      totals={Array []}
    />
  </div>
</span>
`;

exports[`Snapshots AverageTable should render a loading state 1`] = `
<span>
  <div
    style={
      Object {
        "width": "750px",
      }
    }
  >
    <AverageTable
      loading={true}
      totals={Array []}
    />
  </div>
</span>
`;

exports[`Snapshots AverageTable should render the average table 1`] = `
<span>
  <div
    style={
      Object {
        "width": "750px",
      }
    }
  >
    <AverageTable
      dailyData={
        Array [
          Object {
            "day": "1504137600000",
            "metrics": Array [
              Object {
                "diff": 179,
                "label": "Engagement average",
                "value": 1374,
              },
              Object {
                "diff": 43,
                "label": "Impression average",
                "value": 91133,
              },
              Object {
                "diff": 4,
                "label": "Click average",
                "value": 2679,
              },
            ],
          },
          Object {
            "day": "1504224000000",
            "metrics": Array [
              Object {
                "diff": -36,
                "label": "Engagement average",
                "value": 803,
              },
              Object {
                "diff": -7,
                "label": "Impression average",
                "value": 85118,
              },
              Object {
                "diff": -69,
                "label": "Click average",
                "value": 2582,
              },
            ],
          },
          Object {
            "day": "1504310400000",
            "metrics": Array [
              Object {
                "diff": -86,
                "label": "Engagement average",
                "value": 414,
              },
              Object {
                "diff": -42,
                "label": "Impression average",
                "value": 75936,
              },
              Object {
                "diff": -74,
                "label": "Click average",
                "value": 1846,
              },
            ],
          },
          Object {
            "day": "1504396800000",
            "metrics": Array [
              Object {
                "diff": -59,
                "label": "Engagement average",
                "value": 340,
              },
              Object {
                "diff": -18,
                "label": "Impression average",
                "value": 86410,
              },
              Object {
                "diff": -48,
                "label": "Click average",
                "value": 2755,
              },
            ],
          },
          Object {
            "day": "1504483200000",
            "metrics": Array [
              Object {
                "diff": -48,
                "label": "Engagement average",
                "value": 439,
              },
              Object {
                "diff": -12,
                "label": "Impression average",
                "value": 87669,
              },
              Object {
                "diff": -30,
                "label": "Click average",
                "value": 3041,
              },
            ],
          },
          Object {
            "day": "1504569600000",
            "metrics": Array [
              Object {
                "diff": -89,
                "label": "Engagement average",
                "value": 299,
              },
              Object {
                "diff": -22,
                "label": "Impression average",
                "value": 74969,
              },
              Object {
                "diff": -5,
                "label": "Click average",
                "value": 2516,
              },
            ],
          },
          Object {
            "day": "1504656000000",
            "metrics": Array [
              Object {
                "diff": -41,
                "label": "Engagement average",
                "value": 1824,
              },
              Object {
                "diff": 51,
                "label": "Impression average",
                "value": 118827,
              },
              Object {
                "diff": 49,
                "label": "Click average",
                "value": 3933,
              },
            ],
          },
        ]
      }
      loading={false}
      totals={
        Array [
          Object {
            "diff": 25,
            "label": "Impression average",
            "value": 150,
          },
          Object {
            "diff": -55,
            "label": "Click average",
            "value": 1010,
          },
          Object {
            "diff": -39,
            "label": "Engagement average",
            "value": 901,
          },
        ]
      }
    />
  </div>
</span>
`;
